packages/xml-lsp-server/nyc.config.js

```diff
@@ -3,11 +3,12 @@ const nycConfigTypeScript = require("@istanbuljs/nyc-config-typescript");
 module.exports = {
   ...nycConfigTypeScript,
   include: ["src/**/*.ts"],
+  exclude: ["src/server.ts"],
   reporter: ["text", "lcov"],
   "check-coverage": true,
   all: true,
   branches: 100,
   lines: 100,
-  functions: 100,
-  statements: 100
+  functions: 83,
```

Comment:
This conflicts with the guidelines of 100%* coverage for this repo
- https://github.com/SAP/xml-tools/blob/master/CONTRIBUTING.md#test-coverage

We should exclude things that are not covered (assuming there is a good reason)